test/TestApps/ToastNotificationsTestApp/ToastNotificationsTestApp.cpp

```diff
@@ -12,6 +12,11 @@ namespace winrt
     using namespace winrt::Microsoft::Windows::ToastNotifications;
 }
 
+bool BackgroundActivationTest() // Activating application for background test.
+{
+    return true;
```

Comment:
Should you add another "test" to unregister application after doing the background activation?

Reply:
This is actually a great idea. This fixes having to unregister in another packaged test.